doc/nrf/libraries/bootloader/bl_monotonic_counters.rst

```diff
@@ -0,0 +1,47 @@
+.. _doc_bl_monotonic_counters:
+
+Monotonic counters
+##################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The monotonic counters library implements the logic for readind and writing uint16_t counters which can only increase.
```

Comment:
```suggestion
The monotonic counters library implements the logic for reading and writing uint16_t counters which can only increase.
```

Reply:
In an API level reading and updating the value is indeed supported. But I can update it as you suggest, it is no problem.